tests/ProxyManagerTest/Functional/AccessInterceptorScopeLocalizerFunctionalTest.php

```diff
@@ -439,4 +442,35 @@ private function assertProxySynchronized($instance, AccessInterceptorInterface $
             );
         }
     }
+
+    public function testCanCreateAndRegisterCallbackWithVariadicNotation()
+    {
+        if (PHP_VERSION_ID <= 50600) {
+            $this->markTestSkipped('Test can\'t run on < 5.6.0 php version');
+        }
+
+        $configuration = new Configuration();
+        $factory = new AccessInterceptorScopeLocalizerFactory($configuration);
+
+        $targetObject = new ClassWithMethodWithVariadicFunction();
+
+        $object = $factory->createProxy($targetObject, [ function ($paratemers) {
+            return 'Foo Baz';
+        },]);
+
+        $this->assertNull($object->bar);
+        $this->assertNull($object->baz);
+
+        $object->foo('Ocramius', 'Malukenho', 'Danizord');
+        $this->assertSame('Ocramius', $object->bar);
+        $this->assertSame(
```

Comment:
This same assertion should also be replicated in the other proxy types functional tests